app/scripts/filters/timecode.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+	function timecode() {

```

Comment:
Yea I'm not sure why it looks like that. It looks fine on my computer but for whatever reason it's all weird when I push it to github.